### PR TITLE
Updated the documentation to a certain extent

### PR DESCRIPTION
**Pull request by ThatAnonyG**
Updated the documentation for the OP play payload and a few other stuff.

### CODE REVIEW

File: IMPLEMENTATION.md [hunk 1]
```diff
@@ -46,7 +46,11 @@ User-Id: The user id of the bot you are playing music with
 ```
 
 ### Outgoing messages
+
+#### Provides a voice update
```
Comment (freyacodes) [inline IMPLEMENTATION.md]:
```suggestion
#### Provide a voice server update
```
Wrong tense

File: IMPLEMENTATION.md [hunk 2]
```diff
@@ -56,13 +60,17 @@ Provide an intercepted voice server update. This causes the server to connect to
 }
 ```
 
-Cause the player to play a track.
+#### Plays a track
```
Comment (freyacodes) [inline IMPLEMENTATION.md]:
```suggestion
#### Play a track
```
Wrong tense

File: IMPLEMENTATION.md [hunk 3]
```diff
@@ -56,13 +60,17 @@ Provide an intercepted voice server update. This causes the server to connect to
...
 
 `startTime` is an optional setting that determines the number of milliseconds to offset the track by. Defaults to 0.
 
 `endTime` is an optional setting that determines at the number of milliseconds at which point the track should stop playing. Helpful if you only want to play a snippet of a bigger track. By default the track plays until it's end as per the encoded data.
 
-If `noReplace` is set to true, this operation will be ignored if a track is already playing or paused.
+`volume` is an optional setting which determines the default volume of the player.
```
Comment (freyacodes) [inline IMPLEMENTATION.md]:
```suggestion
`volume` is an optional setting which changes the volume if provided.
```

File: IMPLEMENTATION.md [hunk 4]
```diff
@@ -56,13 +60,17 @@ Provide an intercepted voice server update. This causes the server to connect to
...
+
+If `noReplace` is set to true, this operation will be ignored if a track is already playing or paused. This is an optional field.
+
+If `pause` is set to true, the playback will be paused by default. This is an optional field.
```
Comment (freyacodes) [inline IMPLEMENTATION.md]:
```suggestion
If `pause` is set to true, the playback will be paused. This is an optional field.
```

File: IMPLEMENTATION.md [hunk 5]
```diff
@@ -330,15 +353,21 @@ A severity level of `COMMON` indicates that the error is non-fatal and that the
 ```
 
 ---
+
 ### RoutePlanner API
+
 Additionally there are a few REST endpoints for the ip rotation extension
+
 #### RoutePlanner Status
```
Comment (freyacodes) [inline IMPLEMENTATION.md]:
```suggestion
#### Get RoutePlanner status
```

**Review by freyacodes — CHANGES_REQUESTED** [review]

**Comment by ThatAnonyG** [comment]
I accepted all the suggestions. Sorry for the delay in replying.

**Review by freyacodes — APPROVED** [review]
Thanks!

**Pull request merged**